### ISSUE TITLE
Integrate WiLink 8 (L18xxMOD) Single-Band Combo Module (Wifi/BT/BLE)

### ISSUE THREAD

**Issue by frantic0** [BelaPlatform/bela-image#7]
Any walkthrough on how to install? Would it be easy to integrate on the Bela image? It appears that the new BeagleBone Black Wireless also has the same module, it would be interesting to incorporate this.

http://www.ti.com/tool/wilink-sw

**Comment by giuliomoro** [comment]
It should be possible to install it (though it requires to have a copy of the kernel header files handy), but we need to sort out the pin incompatibility of the BBGW first: there would be little point in installing the driver if these wifi modules cannot be used in conjunction with the Bela cape.

**Comment by frantic0** [comment]
Yes, I believe it isn't trivial. I took a peek at this.. Would you say that's the way?

[WiLink8 Linux](http://processors.wiki.ti.com/index.php/WiLink8_Linux_Getting_Started_Guide)
[WL18xx_Build](http://processors.wiki.ti.com/index.php/WL18xx_WiFi_Build_Process#Setting_up_the_Host_Environment)
[WL18xx_Platform_Integration](http://processors.wiki.ti.com/index.php/WL18xx_Platform_Integration_Guide#Appendix_A:_Device_Tree_file_-_patching_and_updating)
[TI_Bluetooth_Stack_for_WL18xx_](http://processors.wiki.ti.com/index.php/TI_Bluetooth_Stack_for_WL18xx_-_Getting_Started_Guide#Installing_Bluetopia_Platform_Manager)

By the way, how's it going with the pin incompatibility? I'd expect that to be further away from trivial.

**Comment by giuliomoro** [comment]
It looks like this driver may be already available on Bela, but the device tree overlay needs to be edited in order for the wifi / bt modules to work AND be able to test the existing driver.

**Comment by frantic0** [comment]
Ok, I'll be looking into that.
